src/main/java/com/microsoft/graph/http/BaseActionCollectionRequestBuilder.java

```diff
@@ -0,0 +1,68 @@
+// ------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation.  All Rights Reserved.  Licensed under the MIT License.  See License in the project root for license information.
+// ------------------------------------------------------------------------------
+
+package com.microsoft.graph.http;
+
+import com.microsoft.graph.core.IBaseClient;
+import com.microsoft.graph.http.BaseRequestBuilder;
+import com.microsoft.graph.options.Option;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * The base method request builder class used for POST actions
```

Comment:
`The base method request builder class used for OData Action POST requests that return a collection.` 

The naming led me to believe that this is bound to a collection, and not referring to the return type.

Reply:
is there an action to take on that?

Reply:
Yeah, the request is to change the comment to proposed comment.

Reply:
```suggestion
 * The base method request builder class used for OData Action POST requests that return a collection.
```

Reply:
like this?